### ISSUE TITLE
[New feature and actual need] How can I have the last 10 position displayed and different color status

### ISSUE THREAD

**Issue by pierrot10** [vitalidze/traccar-web#462]
Hello all

There is a need which will be nice to have particularely for smartphone devise.
It would be nive to have some last devise position of its track.
For exemple, actually from the Prefrences I can  define the Trace Recording Interval, and THEN when I check the box "Record trace", I can see the x last records.

Would it possible, for a devise, to have a checkbox, which enable the check box "record devise" and display the last x record the from the history? Is it easy to adapt it by myself?

Secondely, I really need to make the difference, between the GPS position and converted GSM location to GPS due to difference of precision. I use hdop parameter to mak GSm or GPS position.
I also know, traccar has a field call valid from the positions table but it is not usable. In any case, would it be possible to have, by default, a marker for all GPS position and a maker for GSM position?

Is it easy to adapt it my self before the next version, if you think interessting to add?
If not, could you gibe me some tips in order to ask a Java friend devloppeur to do it, on my traccar version?

many thnak for your help and support and have a nice day.
Cheers

**Comment by pierrot10** [comment]
Dear all, 
Some on can let me know, it point of view?
Would it possible (and how to) to have the last x trace recording interval, displayed by default,  when I load traccar map from a smartphone?

Thank a lot for your support and work

**Comment by vitalidze** [comment]
> Would it possible, for a devise, to have a checkbox, which enable the check box "record devise" and display the last x record the from the history? Is it easy to adapt it by myself?

I don't quite understand. You want to add in mobile verion of UI same function which exists in desktop UI (recording trace)? If so, then it is in plans.

> Secondely, I really need to make the difference, between the GPS position and converted GSM location to GPS due to difference of precision. I use hdop parameter to mak GSm or GPS position.
> I also know, traccar has a field call valid from the positions table but it is not usable. In any case, would it be possible to have, by default, a marker for all GPS position and a maker for GSM position?

I have in plans adding some indication on the map based on sensors value.

> Is it easy to adapt it my self before the next version, if you think interessting to add?
> If not, could you gibe me some tips in order to ask a Java friend devloppeur to do it, on my traccar version?

Currently I don't have any time to work on this project. So probably it's better to ask your friend developer to implement your private tasks.

**Comment by vitalidze** [comment]
> Would it possible (and how to) to have the last x trace recording interval, displayed by default, when I load traccar map from a smartphone?

I think the state of 'record trace' ('selected' or 'not selected'(default)) setting probably may be saved somewhere in user's profile so this can be checked when you log back in.

**Comment by pierrot10** [comment]
Dear Vitalidze,

Thank for your reply.
[quote]
I don't quite understand. You want to add in mobile verion of UI same function which exists in desktop UI (recording trace)? If so, then it is in plans.
[/quote]
When I open traccar with my smartphone, I can see my devise with the mobile web version. I only see it actualy position (same for the desktop version). 
From the mobile version we can not select "record trace".
It would be nice, particularely with the mobile version, to have the previous prosition, additionally to the actual version, AND to have "record trace" active by default.
For exemple, after editing a devise, I would add a check box "Record trace by default" for a devise record.
In that way, when I open a smartphone browser and load traccar interface, I would see the X previous position of a devise which is moving or not. The X previous position are defined in 'Trace Recording Interval' from preferences.
Do you unterstand my suggestion.
Thank a lot

**Comment by vitalidze** [comment]
Yeah, I understand that. Probably it is possible, will add it to my back log.

**Comment by pierrot10** [comment]
Great, thank !

**Comment by tvjp91** [comment]
Hi vitalidze i have a problem 
i configure my platform traccar web UI but something is wrong because event when i set my 
device to send data after 1 minute  traccar shows  data after 10min i don't know why.

**Comment by vitalidze** [comment]
@tvjp91 why are you posting it here? How does it relate with this thread?

Regarding your question, most probably there is some issue with your device. Maybe it does not have GPS fix, or something like this. You need to check tracker-server.log for the messages from your device.
